src/scss/_layout.scss

```diff
@@ -10,6 +10,7 @@
         position: relative;
 
         display: flex;
+        flex-direction: column;
```

Comment:
Are you sure you need to ```flex-direction: column;``` on the ```section``` selector? It will apply to all sections, and you are already using it in your ```.support``` selector.